modules/xcm-bridge-hub/src/backpressure.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
This file should be the copy of the similar primitives (`LocalXcmQueueMessageProcessor` and `LocalXcmQueueSuspender`) from #2294. The only difference is how we "compute" whether the inbound XCM queue should be suspended or not. In v1 PR it looks at the number of queued messages at the outbound lane. Here it should take into account all outbound lanes, opened by the same origin (a bit WIP)